### PR TITLE
Plane: failsafe to use enums instead of magic numbers

### PR DESCRIPTION
**Pull request by magicrub**
This is a non-functional change.

### CODE REVIEW

**Comment by khancyr** [comment]
LGTM

**Comment by rmackay9** [comment]
Re naming, my personal view (which is correct of course) is that we should try and name things from most important to least important.  So instead of "SHORT_FS_ACTION_FBWA" it should probably be "FS_SHORT_ACTION_FBWA" or "FS_ACTION_SHORT_FBWA".

**Comment by magicrub** [comment]
yup, I agree with both of you. Will update

**Comment by magicrub** [comment]
updated

File: ArduPlane/Parameters.h [hunk 1]
```diff
@@ -265,17 +265,17 @@ class Parameters {
...
         k_param_throttle_slewrate,
         k_param_throttle_suppress_manual,
         k_param_throttle_passthru_stabilize,
         k_param_rc_12_old,
         k_param_fs_batt_voltage,
         k_param_fs_batt_mah,
-        k_param_short_fs_timeout,
-        k_param_long_fs_timeout,
+        k_param_fs_timeout_short,
+        k_param_fs_timeout_long,
```
Comment (magicrub) [inline ArduPlane/Parameters.h]: double sanity checking here.. This rename of k_param_* does not matter as long as the order is not changed, right?

Comment (rmackay9) [reply ArduPlane/Parameters.h]: yeah, that's right.  the rename isn't going to cause corruption.

**Pull request merged**